python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -5,7 +5,7 @@
     import unittest
```

Comment:
If we're using `tearDownClass` etc. then this will have to become:
```python
try:
    import unittest2 as unittest
except ImportError:
    import unittest
```
or similar so it can continue to run on Python 2.6.